imports/plugins/core/orders/server/no-meteor/resolvers/Query/ordersByAccountId.js

```diff
@@ -0,0 +1,29 @@
+import { getPaginatedResponse } from "@reactioncommerce/reaction-graphql-utils";
+import { decodeAccountOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/account";
+import { decodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+
+/**
+ * @name Query.ordersByAccountId
+ * @method
+ * @memberof Order/GraphQL
+ * @summary Get an order by its reference ID
+ * @param {Object} parentResult - unused
+ * @param {ConnectionArgs} args - An object of all arguments that were sent by the client
+ * @param {String} args.accountId - accountId of owner of the orders
+ * @param {String} args.shopIds - shop IDs to check for orders from
+ * @param {Object} context - An object containing the per-request state
+ * @return {Promise<Object>|undefined} An Order object
+ */
+export default async function ordersByAccountId(parentResult, args, context) {
+  const { accountId, shopIds: opaqueShopIds } = args;
+
+  const shopIds = opaqueShopIds && opaqueShopIds.map(decodeShopOpaqueId);
+
+  return context.queries.ordersByAccountId(context, {
+    accountId: decodeAccountOpaqueId(accountId),
+    shopIds
+  });
+
+  // TODO: EK - Get paginated response
```

Comment:
@aldeed is this somewhere where we might want paginated responses? Or do we just want to display all orders for a single user?

Reply:
@kieckhafer Yes definitely pagination/connection